tutorials/server/install-and-configure-apache/install-and-configure-apache.md

```diff
@@ -0,0 +1,148 @@
+---
+id: install-and-configure-apache
+summary: This tutorial covers the installation and configuration of an Apache web server
+categories: server
+tags: ubuntu-server, apache, web server
+difficulty: 3
+status: published
+feedback-url: https://github.com/canonical-websites/tutorials.ubuntu.com/issues
+author: Aden Padilla <adenpadilla@gmail.com>
+published: 2017-11-30
+
+---
+
+# Install and Configure Apache
+
+## Overview
+Duration: 1:00
+
+Apache is an open source web server that's available for Linux servers free of charge.
+
+In this tutorial we'll be going through the steps of setting up an Apache server.
+
+### What you'll learn
+- How to set up Apache
+- Some basic Apache configuration
+
+### What you'll need
+- Ubuntu Server 16.04 LTS
+- Secure Shell (SSH) access to your server
+- Basic Linux command line knowledge
+
+Got everything ready? Let's move on to the next step!
+
+## Installing Apache
+Duration: 1:00
+
+To install Apache, install the latest meta-package `apache2` by running:
+
+```bash
+sudo apt update
+sudo apt install apache2
+```
+
+After letting the command run, all required packages are installed and we can test it out by typing in our IP address for the web server.
+
+
+![Apache-Installed](images/install-success.png)
+
+If you see the page above, it means that Apache has been successfully installed on your server! Let's move on.
+
+## Creating Your Own Website
+Duration: 4:00
+
+By default, Apache comes with a basic site (the one that we saw in the previous step) enabled. We can modify its content in `/var/www/html` or settings by editing its Virtual Host file found in `/etc/apache2/sites-enabled/000-default.conf`.
+
+We can modify how Apache handles incoming requests and have multiple sites running on the same server by editing its Virtual Hosts file.
+
+Today, we're going to leave the default Apache virtual host configuration pointing to www.example.com and set up our own at gci.example.com.
+So let's start by creating a folder for our new website in `/var/www/` by running
```

Comment:
There needs to be a carriage return between these two paragraphs. Also, even though this wasn't mentioned before, it might be worth making the web addresses bold (but putting double asterisks \** **either side** \**).

Reply:
you're right, it does look better that way!